yt_dlp/extractor/radiko.py

```diff
@@ -33,6 +34,8 @@ class RadikoBaseIE(InfoExtractor):
     _HOSTS_FOR_LIVE = (
         'https://c-radiko.smartstream.ne.jp',
     )
+    # performer separtor character (include Japanese charset)
+    _PERFORMER_SPLIT_RE = "[/／、　,，]"
```

Comment:
the separator character class is a good idea, but we can inline this in the method
```suggestion
```